### DISCUSSION TITLE
Issue Sending OTP with email_auth package

### DISCUSSION THREAD

**Discussion by Virtue-226** [saran-surya/email_auth#64] [Q&A] [answered]
I get this error trying to send an OTP
Failed to send OTP ❌
I/flutter (28668): email-auth >> Message from server :: (534, b'5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbs\n5.7.14 dO4NsUvDQtOvIPkQIm8JisFI2umRg9nkB_r_UjMdZO6cYNFGSs-C7U2qI6q52Z2svJA0H\n5.7.14 PjY01M38SSkRolxQeWpMe_-xBx8RYjBvgXLPZl2GAZU2ORz6jwJ-UnzvV4orsLEt>\n5.7.14 Please log in via your web browser and then try again.\n5.7.14  Learn more at\n5.7.14  https://support.google.com/mail/answer/78754 bs16sm22284533qkb.69 - gsmtp')
I/flutter (28668): Server error cannot send mail : section 1
That;s my debug log
Please what can i do?

**Comment by saran-surya** [comment, marked as answer]
This is a production server issue, Seems like You have not enabled 
## Allow Less Secure Apps,
Kindly read the docs, https://saran-surya.github.io/email-auth-node/ "Step : 3"

**Reply by Virtue-226** [reply]
Please i have enabled allow less secure apps for the Gmail i created.
Am still confused why is not working.
Please what should i do?

**Reply by Virtue-226** [reply]
It is working now.
Thanks

**Comment by Virtue-226** [comment]
please i got this error trying to send otp today.
email-auth >> Remote server is not available -- using test server --
I/flutter ( 9528): email-auth >> ❗ Warning this is not reliable on production
I don't know why this issue....

**Reply by saran-surya** [reply]
Have you passed the config file ?
